data/successor 1 introduction.txt

```diff
@@ -871,9 +872,11 @@ mission "Successors: Trusted"
 			choice
 				`	"Indeed. I'm ready to begin, then."`
 					goto start
+				`	"Alright. Let's go."`
+					goto start
 			label start
 			`	"Good! We would be wise to be early to our respective events."`
-			`	Sieasej brings you to a concierge of sorts, who eventually informs you that you should make your way to one of the smaller rooms branching off from the lobby. The doors open to reveal a garden-like area, filled with large long-leafed plants surrounding a centrally-located depression half-filled with water. Several Successors, many well-dressed like before, are gathered in the pool; you can't help but wince a little as the frigid water floods your boots.`
+			`	Sieasej brings you to a concierge of sorts, who eventually informs you that you should make your way to one of the smaller rooms branching off from the lobby. The doors open to reveal a garden-like area, filled with large long-leafed plants surrounding a centrally-located depression half-filled with water. Several Successors, many well-dressed like before, are gathered in the pool; but you can't help but wince a little as the frigid water floods your boots.`
```

Comment:
This sentence could (and perhaps should) be reworded to not use a semicolon here, but as is, I don't think the "but" here is necessary.